### DISCUSSION TITLE
pgsc_calc v2.0.0-alpha.3

### DISCUSSION THREAD

**Discussion by nebfield** [PGScatalog/pgsc_calc#193] [Announcements]
# Improvements:

* Automatically retry scoring with more RAM on larger datasets
* Describe scoring precision in docs 
* Change handling of VCFs to reduce errors when recoding 
* Internal changes to improve support for custom reference panels

# Bug fixes:

* Fix VCF input to ancestry projection subworkflow (thanks [`frahimov`](https://github.com/PGScatalog/pgsc_calc/issues/172
) and [`AWS-crafter`](https://github.com/PGScatalog/pgsc_calc/issues/155) for patiently debugging)
* Fix scoring options when reading allelic frequencies from a reference panel (thanks [`raimondsre`](https://github.com/PGScatalog/pgsc_calc/pull/139#issuecomment-1736313211) for reporting the changes from v1.3.2 -> 2.0.0-alpha)
* Fix conda profile action

<hr /><em>This discussion was created from the release <a href='https://github.com/PGScatalog/pgsc_calc/releases/tag/v2.0.0-alpha.3'>pgsc_calc v2.0.0-alpha.3</a>.</em>

**Comment by smlmbrt** [comment]
This is a **critical update**, as scoring without `--run_ancestry` will give erroneous results if the genotyped variants have high missingness.
